src/coreclr/jit/emitarm64.cpp

```diff
@@ -17330,6 +17714,31 @@ void emitter::emitDispExtendOpts(insOpts opt)
         assert(!"Bad value");
 }
 
+/*****************************************************************************
+ *
+ *  Prints the encoding for the Extend Type encoding
+ */
+
+void emitter::emitDispSveExtendOpts(insOpts opt)
+{
+    switch (opt)
+    {
+        case INS_OPTS_SCALABLE_S_UXTW:
+        case INS_OPTS_SCALABLE_D_UXTW:
+            printf("UXTW");
```

Comment:
Make these prints lower case to match capstone?

Reply:
I can make them lower case